aptos-move/aptos-vm/src/transaction_metadata.rs

```diff
@@ -31,6 +33,7 @@ pub struct TransactionMetadata {
     pub script_hash: Vec<u8>,
     pub script_size: NumBytes,
     pub is_keyless: bool,
+    // Question[Orderless]: Why are we storing entry function payload here, but not script payload?
```

Comment:
This is for some native functions which allow you to get current entry function name from Move code.

Reply:
Gotcha.